src/keyszer/lib/key_context.py

```diff
@@ -28,4 +28,13 @@ def x_error(self):
 
     @property
     def device_name(self):
+        self._device_name = self.device.name
         return self._device_name
+
+    @property
+    def capslock_on(self):
+        return True if 1 in self.device.leds() else False
```

Comment:
We should pull the 1 and 0 out into constants probably instead of magic numbers. <https://www.codebyamir.com/blog/software-anti-patterns-magic-numbers>

Reply:
Howzis? Or should the constants be set in `init`? 

```python
    @property
    def capslock_on(self):
        CL_LED = 1
        return True if CL_LED in self.device.leds() else False

    @property
    def numlock_on(self):
        NL_LED = 0
        return True if NL_LED in self.device.leds() else False
```